wallet/src/ui/app/redux/RootReducer.ts

```diff
@@ -7,8 +7,16 @@ import account from './slices/account';
 import app from './slices/app';
 import suiObjects from './slices/sui-objects';
 import transactions from './slices/transactions';
+// TODO add to transactions slice
+import txresults from './slices/txresults';
```

Comment:
I don't think we need a new slice, transactions slice is the one we should use to store all the transactions of the user.

Reply:
OK